packages/application-shell/src/components/login/login.js

```diff
@@ -118,210 +92,152 @@ export class Login extends React.PureComponent {
+                  touched={formikProps.touched.email}
+                  errors={formikProps.errors.email}
+                  onChange={formikProps.handleChange}
+                  onBlur={formikProps.handleBlur}
+                  isDisabled={formikProps.isSubmitting}
+                  renderError={this.renderEmailErrors}
+                />
+                <TextField
+                  name="password"
+                  title={this.props.intl.formatMessage(messages.password)}
+                  isRequired={true}
+                  value={formikProps.values.password}
+                  touched={formikProps.touched.password}
+                  errors={formikProps.errors.password}
+                  onChange={formikProps.handleChange}
+                  onBlur={formikProps.handleBlur}
+                  isDisabled={formikProps.isSubmitting}
+                />
+                <Spacings.Inline>
+                  {/* FIXME: the <PrimaryButton> current does not behave like a
+                  "submit" button, making it impossible to handle "submit on enter".
+                  Once that is fixed in the underlying component, the "submit on enter"
+                  will work. */}
+                  <PrimaryButton
+                    label={
+                      this.state.loading
+                        ? this.props.intl.formatMessage(messages.validating)
+                        : this.props.intl.formatMessage(messages.signin)
+                    }
+                    onClick={formikProps.handleSubmit}
+                    isDisabled={formikProps.isSubmitting}
                   />
-                </Spacings.Stack>
-              </Spacings.Stack>
+                </Spacings.Inline>
 
-              <div>
-                <PrimaryButton
-                  label={
-                    this.state.loading
-                      ? this.props.intl.formatMessage(messages.validating)
-                      : this.props.intl.formatMessage(messages.signin)
+                <Countdown
+                  onCountEnd={() =>
+                    redirect(`${this.props.adminCenterUrl}/reset-password`)
                   }
-                  onClick={this.handleSubmit}
-                />
-              </div>
-
-              <a
-                className={classnames(styles['forgot-pw'], {
-                  [styles.disabled]: this.state.loading,
-                })}
-                data-track-component="ForgotPassword"
-                data-track-event="click"
-                onClick={event =>
-                  !this.state.loading
-                    ? this.redirectToForgotPassword(event)
-                    : null
-                }
-              >
-                <FormattedMessage {...messages.forgotPassword} />
-              </a>
-            </Spacings.Stack>
-          </fieldset>
-        </form>
-      </LoginBox>
+                >
+                  {({ handleClick }) => (
+                    <a
```

Comment:
does it make sense to make this a `<a>` if it isn't a link? could we make it a <p> or a <span> instead?

Reply:
It will be a link eventually, once the forgot password page is implemented in the MC. So I would keep it like this for now